.goreleaser.yaml

```diff
@@ -0,0 +1,59 @@
+version: 2
+
+before:
+  hooks:
+    - go mod tidy
+    - go generate ./...
+
+builds:
+  - env:
+      - CGO_ENABLED=0
+
+kos:
+  - repositories:
+      - ghcr.io/barakplasma/whatsmeow-transcribe
+      # - ghcr.io/hoehermann/whatsmeow-transcribe
```

Comment:
you uncomment this, and comment out the line above it to build a docker image and push it to your repo